docs/config.json

```diff
@@ -368,8 +368,8 @@
       "timeline": [
         {
           "moved": 0,
-          "ringId": "adopt",
-          "date": "2022-05-01",
+          "ringId": "hold",
+          "date": "2022-12-19",
           "description": ""
         }
```

Comment:
Oh interesting! It wasn't clear to me from the README that that is how we should use the `entries` list. I'll see about clarifying that in the README too.

> BTW is it worth putting any description in?
Probably. I can add.

Reply:
Fixed timeline with 1a003e742ad0a0ccc2f8803fae98fdc7eed65855. Note I decided to leave `moved` at `0` because we're not actively trying to move off Nginx. I'd say we're just holding on it.

Reply:
I added to the README here: d8a8e35d2a34f08b8cd037687b132e43d46bd408

Reply:
I was overloading / mis-remembering the purpose of 'moved'... good call, I agree since we're not actively trying to remove it, that it can stay at 0.

Reply:
Also thanks for adding the context to the readme! 💯